README.md

```diff
@@ -56,6 +55,49 @@ yarn start --sources 'packages/esm-patient-biometrics-app' --sources 'packages/e
 
 Alternatively, you could run `yarn serve` from within the individual packages and then use [import map overrides](http://o3-dev.docs.openmrs.org/#/getting_started/setup?id=import-map-overrides).
 
+## Running tests
+
+To run tests for all packages, run:
+
+```bash
+yarn turbo test
+```
+
+To run tests in `watch` mode, run:
+
+```bash
+yarn turbo test:watch
+```
+
+To generate a `coverage` report, run:
+
+```bash
+yarn turbo coverage
+```
+
+To run tests for a specific package, run:
+
+```bash
+yarn turbo test --filter="<package-name>"
+
+# For example, to run tests for just the Conditions app, run:
+yarn turbo test --filter="esm-patient-conditions-app"
+```
```

Comment:
Filtering by package is probably a more frequent task than generating coverage.

```suggestion
To run tests for a specific package, run:

```bash
yarn turbo test --filter="<package-name>"

# For example, to run tests for just the Conditions app, run:
yarn turbo test --filter="esm-patient-conditions-app"
```

To generate a `coverage` report, run:

```bash
yarn turbo coverage
```
```